Runtime/src/index.ts

```diff
@@ -127,28 +121,28 @@ export class SwiftRuntime {
             payload1: number,
             payload2: number
         ) => {
-            const obj = this.memory.getObject(ref);
-            const key = this.memory.getObject(name);
-            const value = JSValue.decode(kind, payload1, payload2, this.memory);
+            const memory = this.memory;
+            const obj = memory.getObject(ref);
+            const key = memory.getObject(name);
+            const value = JSValue.decode(kind, payload1, payload2, memory);
             obj[key] = value;
         },
         swjs_get_prop: (
```

Comment:
Would you mind adding doc comments to functions in `wasmImports`, including their parameters and return values/types? Thanks!

Reply:
They would be exactly same as `_CJavaScriptKit.h` one. Do you mean we need to add docs here as well for library users?

Reply:
Ah, makes sense. Then we probably don't need those duplicated I think. Thanks for checking!